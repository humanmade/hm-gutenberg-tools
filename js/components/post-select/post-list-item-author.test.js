import React from 'react';
import TestRenderer from 'react-test-renderer';

import PostListItemAuthor from './post-list-item-author';

const author = {
	name: 'Matt Mullenweg',
}

test( 'Post List Item Author', () => {
	const testRenderer = TestRenderer.create( <PostListItemAuthor id={ 1 } author={ author } /> );
	expect( testRenderer.toJSON() ).toMatchSnapshot();
} );
