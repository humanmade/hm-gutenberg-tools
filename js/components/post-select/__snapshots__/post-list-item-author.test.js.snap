// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post List Item Author 1`] = `
<span>
  <b>
    Author:
  </b>
   
  Matt Mullenweg
</span>
`;
